modules/gitgraph/graph_test.go

```diff
@@ -270,446 +270,446 @@ func TestCommitStringParsing(t *testing.T) {
-|/| | | | 
-| |/ / /  
-| * | |   
-| |\ \ \  
-| | * | | 
-| |/| | | 
-| | |/ /  
-| | * |   
-| | |\ \  
-| | | * | 
-| | |/| | 
-* | | | | 
-* | | | |   
-|\ \ \ \ \  
-| * | | | | 
-|/| | | | | 
-| | * | | | 
-| | * | | | 
-| | * | | | 
-| |/ / / /  
-| * | | |   
-| |\ \ \ \  
-| | * | | | 
-| |/| | | | 
-* | | | | | 
-* | | | | | 
-* | | | | | 
-* | | | | | 
-* | | | | | 
-| | | | * | 
-* | | | | |   
-|\ \ \ \ \ \  
-| * | | | | | 
-|/| | | | | | 
-| | | | | * | 
-| | | | |/ /  
-* | | | | |   
-|\ \ \ \ \ \  
-* | | | | | | 
-* | | | | | | 
-| | | | * | | 
-* | | | | | | 
-* | | | | | |   
-|\ \ \ \ \ \ \  
-| | |_|_|/ / /  
-| |/| | | | |   
-| | | | * | | 
-| | | | * | | 
-| | | | * | | 
-| | | | * | | 
-| | | | * | | 
-| | | | * | | 
-| | | |/ / /  
-| | | * | | 
-| | | * | | 
-| | | * | | 
-| | |/| | | 
-| | | * | | 
-| | |/| | | 
-| | | |/ /  
-| | * | | 
-| |/| | | 
-| | | * | 
-| | |/ /  
-| | * | 
-| * | |   
-| |\ \ \  
-| * | | | 
-| | * | | 
-| |/| | | 
-| | |/ /  
-| | * |   
-| | |\ \  
-| | * | | 
-* | | | | 
-|\| | | | 
-| * | | | 
-| * | | | 
-| * | | | 
-| | * | | 
-| * | | | 
-| |\| | | 
-| * | | | 
-| | * | | 
-| | * | | 
-| * | | | 
-| * | | | 
-| * | | | 
-| * | | | 
-| * | | | 
-| * | | | 
-| * | | | 
-| * | | | 
-| | * | | 
-| * | | | 
-| * | | | 
-| * | | | 
-| * | | | 
-| | * | | 
-* | | | | 
-|\| | | | 
-| | * | | 
-| * | | | 
-| |\| | | 
-| | * | | 
-| | * | | 
-| | * | | 
-| | | * | 
-* | | | | 
-|\| | | | 
-| | * | | 
-| | |/ /  
-| * | | 
-| * | | 
-| |\| | 
-* | | | 
-|\| | | 
-| | * | 
-| | * | 
-| | * | 
-| * | | 
-| | * | 
-| * | | 
-| | * | 
-| | * | 
-| | * | 
-| * | | 
-| * | | 
-| * | | 
-| * | | 
-| * | | 
-| * | | 
-| * | | 
-* | | | 
-|\| | | 
-| * | | 
-| |\| | 
-| | * |   
-| | |\ \  
-* | | | | 
-|\| | | | 
-| * | | | 
-| |\| | | 
-| | * | | 
-| | | * | 
-| | |/ /  
-* | | | 
-* | | | 
-|\| | | 
-| * | | 
-| |\| | 
-| | * | 
-| | * | 
-| | * | 
-| | | * 
-* | | | 
-|\| | | 
-| * | | 
-| * | | 
-| | | *   
-| | | |\  
-* | | | | 
-| |_|_|/  
-|/| | |   
-| * | | 
-| |\| | 
-| | * | 
-| | * | 
-| | * | 
-| | * | 
-| | * | 
-| * | | 
-* | | | 
-|\| | | 
-| * | | 
-|/| | | 
-| |/ /  
-| * |   
-| |\ \  
-| * | | 
-| * | | 
-* | | | 
-|\| | | 
-| | * | 
-| * | | 
-| * | | 
-| * | | 
-* | | | 
-|\| | | 
-| * | | 
-| * | | 
-| | * |   
-| | |\ \  
-| | |/ /  
-| |/| |   
-| * | | 
-* | | | 
-|\| | | 
-| * | | 
-* | | | 
-|\| | | 
-| * | |   
-| |\ \ \  
-| * | | | 
-| * | | | 
-| | | * | 
-| * | | | 
-| * | | | 
-| | |/ /  
-| |/| |   
-| | * | 
-* | | | 
-|\| | | 
-| * | | 
-| * | | 
-| * | | 
-| * | | 
-| * | |   
-| |\ \ \  
-* | | | | 
-|\| | | | 
-| * | | | 
-| * | | | 
-* | | | | 
-* | | | | 
-|\| | | | 
-| | | | *   
-| | | | |\  
-| |_|_|_|/  
-|/| | | |   
-| * | | | 
-* | | | | 
-* | | | | 
-|\| | | | 
-| * | | |   
-| |\ \ \ \  
-| | | |/ /  
-| | |/| |   
-| * | | | 
-| * | | | 
-| * | | | 
-| * | | | 
-| | * | | 
-| | | * | 
-| | |/ /  
-| |/| |   
-* | | | 
-|\| | | 
-| * | | 
```

Comment:
unrelated

Reply:
Interesting. I have configured my Vim to strip out trailing spaces?
Why is there trailing space in the first place?
But I can remove it of course.